themes/default/content/registry/packages/aws/installation-configuration.md

```diff
@@ -73,36 +61,18 @@ aws_access_key_id = <YOUR_ACCESS_KEY_ID>
 aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>
 ```
 
-If you want to specify multiple profiles, those are listed in different sections:
-
-```ini
-[default]
-aws_access_key_id = <YOUR_DEFAULT_ACCESS_KEY_ID>
-aws_secret_access_key = <YOUR_DEFAULT_SECRET_ACCESS_KEY>
-
-[test-account]
-aws_access_key_id = <YOUR_TEST_ACCESS_KEY_ID>
-aws_secret_access_key = <YOUR_TEST_SECRET_ACCESS_KEY>
-
-[prod-account]
-aws_access_key_id = <YOUR_PROD_ACCESS_KEY_ID>
-aws_secret_access_key = <YOUR_PROD_SECRET_ACCESS_KEY>
-```
-
-In this case, you will need to set the `AWS_PROFILE` environment variable to the name of the profile to use.
-
-### Set environment variables
+### Set credentials as environment variables
```

Comment:
I think I'd suggest putting environment variables first. One reason is just that it's the simplest thing to do and understand, but also, putting shared-credentials second leads nicely into profiles option, which is an extension of that same idea. (And should work well with your "as an optional step" wording too.)

Reply:
I actually debated putting them first, so I appreciate having you weigh in on this!